doc/supportedPlatformVerions.md

```diff
@@ -189,10 +196,104 @@
 - 3.8.6
 - 3.9.0b1
 - 3.9.0
+- 3.9.1rc1
+- 3.10.0a2
 
-## ruby
+### ruby
 
 - 2.7.1
 - 2.6.6
 - 2.5.8
 
+## Supported buster versions
+
+### dotnet
+
+- 2.1.811
```

Comment:
we need to double check whether the mentioned versions are available as platform binaries ...

Reply:
yes, I checked it's consistent with the available ones in our storage account